components/WhyMeditate/WhyMeditate.module.css

```diff
@@ -0,0 +1,105 @@
+.contentContainer {
+  min-height: 100%;
+  background: url(/buddha-bg2.jpg);
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-attachment: fixed;
+}
+.title {
+  margin: 0;
+  padding: 2% 4%;
+}
+.reason {
+  max-width: 620px;
+  height: 48px;
+  padding: 0 4%;
+}
+.audioContainer {
+  background-color: #fffffffa;
+  border-radius: 50%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  background-clip: content-box;
+  overflow: hidden;
+}
+
+.audioContainer.isPlayed {
+  left: 0;
+  top: 0;
+
+  border: 15px solid rgba(0, 0, 0, 0);
+}
+
+.playerContainer {
+  position: relative;
+  margin-left: 5%;
```

Comment:
This margin is way too big. I think 2% is enough. Especially before playing. Also this needs a bit more margin top.